README.md

```diff
@@ -341,6 +347,7 @@ Table of Contents
   * [codefresh.io](https://codefresh.io) — Free-for-Life plan: 1 build, 1 environment, shared servers, unlimited public repos
   * [codeship.com](https://codeship.com/) — 100 private builds/month, 5 private projects, unlimited for Open Source
   * [deployhq.com](https://www.deployhq.com/) — 1 project with 10 daily deployments (30 build minutes/month)
+  * [GitHub Actions](https://github.com/features/actions) - Free for public repos, 2000 build minutes/month for private repos. Supports Windows, Linux, OSX, ARM, with custom containers
```

Comment:
same basic comment here, github is already listed and it mentions CI/CD (actions)

Reply:
Done